### PR TITLE
Modify ParquetTpchTest to save TpchTables in SetUpTestSuite

### PR DESCRIPTION
**Pull request by majetideepak**
ParquetTpchTest.xxx tests are taking > 20 minutes to run. 
All the time is being spent in writing TPC-H data to a DuckDB table. This happens for each of the 19 tests.
Reduce the execution time by writing the DuckDB table once for the entire test suite.

Resolves https://github.com/facebookincubator/velox/issues/7855

### CODE REVIEW

**Review by mbasmanova — COMMENTED** [review]
@Yuhta Jimmy, would you help review this PR?

**Comment by majetideepak** [comment]
@pedroerp, @Yuhta can you review this PR. We can knock off about 30 minutes of CI time.
https://app.circleci.com/pipelines/github/facebookincubator/velox/39926/workflows/5f7abfd4-26a7-45a0-babe-29d7548c7109

File: velox/dwio/parquet/tests/ParquetTpchTest.cpp [hunk 1]
```diff
@@ -147,13 +146,17 @@ class ParquetTpchTest : public testing::Test {
...
+  static std::shared_ptr<DuckDbQueryRunner> duckDb_;
+  static std::shared_ptr<TempDirectoryPath> tempDirectory_;
+  static std::shared_ptr<TpchQueryBuilder> tpchBuilder_;
 
   static constexpr char const* kTpchConnectorId{"test-tpch"};
 };
 
+std::shared_ptr<DuckDbQueryRunner> ParquetTpchTest::duckDb_ = nullptr;
+std::shared_ptr<TempDirectoryPath> ParquetTpchTest::tempDirectory_ = nullptr;
+std::shared_ptr<TpchQueryBuilder> ParquetTpchTest::tpchBuilder_ = nullptr;
```
Comment (pedroerp) [inline velox/dwio/parquet/tests/ParquetTpchTest.cpp:158]: is this necessary? shared-ptr are always initialized with nullptr by default.

**Review by pedroerp — APPROVED** [review]
Looks good to me, thanks!

**Comment by pedroerp** [comment]
FYI, our internal CI system at Meta runs tests in different processes by default, so this won't have any effect there, but will for sure help our open source CI.

File: velox/dwio/parquet/tests/ParquetTpchTest.cpp (hunk 1 above)
Comment (majetideepak) [reply velox/dwio/parquet/tests/ParquetTpchTest.cpp:158]: probably not. I will cleanup in a follow-up PR.

**Comment by majetideepak** [comment]
> FYI, our internal CI system at Meta runs tests in different processes by default, so this won't have any effect there, but will for sure help our open source CI.

We can improve Velox vector to DuckDB vector conversion. It is row-oriented now, hence slow.

**Pull request closed without merging**